services/drupal/web/modules/custom/epa_workflow/src/Plugin/Block/EpaNodeDetailsBlock.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Drupal\epa_workflow\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+
+/**
+ * Provides an EPA Node Details block.
+ *
+ * @Block(
+ *   id = "epa_node_details",
+ *   admin_label = @Translation("EPA Node Details"),
+ *   category = @Translation("Custom"),
+ *   context_definitions = {
+ *     "node" = @ContextDefinition("entity:node", required = TRUE, label = @Translation("Node"))
+ *   }
+ * )
+ */
+class EpaNodeDetailsBlock extends BlockBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * Constructs a new EpaNodeDetailsBlock instance.
+   *
+   * @param array $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin_id for the plugin instance.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager
```

Comment:
Adjust the `entityTypeManager` variable here to be in `snake_case` rather than `camelCase`